src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -19,39 +19,48 @@ In this mode, the CLI can only interact with one project at a time.
 # API Keys {% #api-keys %}
 In non-interactive mode, the CLI uses an API key to authenticate. Your API key must have sufficient permissions to execute the commands you plan to use. [Learn more about API Keys](/docs/advanced/platform/api-keys).
 
+# Providers {% #providers %}
+
+In non-interactive mode, you can use providers like [Github actions](https://github.com/appwrite/setup-for-actions/tree/feat-cli-g2?tab=readme-ov-file#introduction) to automate your Appwrite CLI commands, allowing you to use the CLI commands even in non-interactive mode.
+
```

Comment:
Providers is very vague. Let's move this to the end and rename the heading to CI/CD integrations. Introduce the section and then under that, we can have a heading for GitHub and then talk about the GitHub action.